test/functional/rubrics_controller_test.rb

```diff
@@ -119,6 +119,17 @@ class RubricsControllerTest < AuthenticatedControllerTest
       assert_response :redirect
     end
 
+    should 'deal properly with malformed CSV files' do
+      tempfile = fixture_file_upload('files/malformed.csv')
+      post_as @admin,
+              :csv_upload,
+              :assignment_id => @assignment.id,
+              :csv_upload => {:rubric => tempfile}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.